src/components/Catalogue/catalogue.tsx

```diff
@@ -0,0 +1,120 @@
+  const location = useLocation();
+  const totalPages = 6;//temporarily hardcoded
+
+  const fetchItems = async (page: number) => {
+    //request is under,this is just to display styling
+    try {
+      const response = await axios.get<Photo[]>(`https://jsonplaceholder.typicode.com/photos?_page=${page}&_limit=${PAGE_SIZE}`);
+      // const response = await agent.Catalogue.fetchItems(page, PAGE_SIZE);
+      setItems(response.data);
+      console.log(response.data, 'Items fetched succesfully')
+    } catch (error) {
+      console.error('Error fetching items:', error);
+    }
+  };
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const pageParam = searchParams.get('page');
+    const pageNumber = pageParam ? parseInt(pageParam, 10) : 1;
+    if (pageNumber >= 1 && pageNumber <= totalPages) {
+      setCurrentPage(pageNumber);
+    } else {
+      setNotFound(true);
+    }
+  }, [location.search]);
+
+  useEffect(() => {
+    fetchItems(currentPage);
+  }, [currentPage, navigate]);
+
+  const paginate = (pageNumber: number) => {
+    if (pageNumber >= 1 && pageNumber <= totalPages) {
+      setCurrentPage(pageNumber);
+      navigate(`?page=${pageNumber}`);
+    }
+  };
+
+  return (<>{
+    notFound ? <NotFound /> : <div className="catalogue-container" >
+      <div className='cat_left'>
+        <div className='cat_filter'>
+          <div className='cat_photo'>
+            <h6>Фото</h6>
+            <SwitchBtn />
+          </div>
+          <div className='filter_categories'>
+            <ul>
+              <li>
+                <img src="" />
+                <h6>Категорія</h6>
+                <img src="/icons/filter_expand.svg" />
+              </li>
+              <li>
+                <img src="" />
+                <h6>Місце</h6>
+                <img src="/icons/filter_expand.svg" />
+              </li>
+              <li>
+                <img src="" />
+                <h6>Матеріал</h6>
+                <img src="/icons/filter_expand.svg" />
+              </li>
+              <li>
+                <img src="" />
+                <h6>Дата створення</h6>
+                <img src="/icons/filter_expand.svg" />
+              </li>
+              <li>
+                <img src="" />
+                <h6>Статус</h6>
+                <img src="/icons/filter_expand.svg" />
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div className='cat_right'>
+        <Search />
+        <div className="cat-items-container">
+          {items && items.map((item,index) => (
+            <Link key={item.id} to={`/catalogue/${item.id}`} className="cat-item">
+              <img src={item.thumbnailUrl} alt={item.title} />
+              {/* <div className='cat-item-title'><a>{item.title}</a></div> */}
+              {index === 0 ? <a>Мозаїчне зображення Димитрія Солунського</a> : <a>{item.title}</a>}
+            </Link>
+          ))}
```

Comment:
I guess `<a>` on line 107 was placed by accident cause it doesn't even have any forwarding. Better to remove it cause DOM doesn't like nesting `<a>`